nucypher/blockchain/eth/token.py

```diff
@@ -782,16 +784,16 @@ def __fire_commitment(self):
         transacting_power = self.worker.transacting_power
         with transacting_power:
             txhash = self.worker.commit_to_next_period(fire_and_forget=True)  # < --- blockchain WRITE
-        self.log.info(f"Making a commitment to period {self.current_period} - TxHash: {txhash.hex()}")
-        return txhash
+            self.log.info(f"Making a commitment to period {self.current_period} - TxHash: {txhash.hex()}")
+            return txhash
```

Comment:
Isn't it preferable to remain in the context only for the transacting operation? Just wondering what's the motivation for doing it differently.